src/Core/Components/DataGrid/FluentDataGrid.razor.css

```diff
@@ -2,18 +2,18 @@ fluent-data-grid {
     --col-gap: 1rem;
 }
 
-::deep .empty-content-row {
+::deep .empty-content-row,
```

Comment:
You should have `fluent-data-grid` before each sub-classed, to avoid conflicts if the dev use another class with same name (.empty-content-cell, ...)

Reply:
Nope, because it is an isolated css, this is already scoped to this component only